### DISCUSSION TITLE
How to configure TypeDORM to work with local dynamoDB

### DISCUSSION THREAD

**Discussion by PabloArrietaL** [typedorm/typedorm#79] [Q&A] [answered]
How can I configure typedORM to work with local and remote dynamodb?

**Comment by whimzyLive** [comment, marked as answer]
Hi @PabloArrietaL 
Each connection your create accepts optional `documentClient` instance from aws-sdk which is used by managers to communicate with dynamodb, so you can simply provide an documentClient with local config like this:

```typescript

    const myConnection = createConnection({
      documentClient: new DocumentClient({
        endpoint: 'http://localhost:3000', // or this can be any other endpoint
        credentials: {
          accessKeyId: 'ACCESS KEY ID',
          secretAccessKey: 'SECRET ACCESS KEY'
        }
      })
    })
```

Please let me know if this has helped you, so this can be converted into a discussion for better visibility.
